src/makeCalculator.js

```diff
@@ -36,8 +36,45 @@
  *
  * @return {object}
  */
+
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(callback, digit) {
+      this.result = callback(this.result, digit);
+
+      return this;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    add(param1, param2) {
+      return param1 + param2;
```

Comment:
Maybe better to change parameters names like startParameter fot first parameter and digit or enteredDigit for second?